goclean.sh

```diff
@@ -11,21 +11,14 @@
 
 set -ex
 
-# Make sure glide is installed and $GOPATH/bin is in your path.
```

Comment:
The script doesn't rely on glide nor dep. So I thought it was not necessary to explicitly check if dep was in the gopath.